core/resources/src/main/res/values/themes.xml

```diff
@@ -27,7 +27,7 @@
         <item name="android:windowLightNavigationBar" tools:targetApi="27">true</item>
     </style>
 
-    <style name="Theme.GoCart.SplashScreen" parent="NightAdjusted.Theme.Splash">
+    <style name="Theme.GoCart.SplashScreen" parent="Theme.SplashScreen">
```

Comment:
We can safely revert back to `NightAdjusted.Theme.SplashScreen` since the background color is changed on dark theme